features/steps/promotions_steps.py

```diff
@@ -32,9 +38,79 @@ def step_impl(context):
             "name": row['name'],
             "start_date": row['start_date'],
             "end_date": row['end_date'],
-            "whole_store": row['whole_store'] in ['True', 'False'],
+            "whole_store": row['whole_store'] in ['True', 'true', '1'],
             "message": row['message'],
-            "promotion_changes_price": row['promotion_changes_price'] in ['True', 'False']
+            "promotion_changes_price": row['promotion_changes_price'] in ['True', 'true', '1'],
+            "has_been_extended": row['promotion_changes_price'] in ['True', 'true', '1'],
+            "original_end_date": row['end_date'],
         }
         context.resp = requests.post(rest_endpoint, json=payload)
         assert(context.resp.status_code == HTTP_201_CREATED)
+
+
+@given(u'the server is started')
```

Comment:
I'm not sure, but I saw these lines (51-116) in web_steps.py in Professor's lab.

Reply:
I don't think it makes a difference.